### DISCUSSION TITLE
wagmi 0.7.x?

### DISCUSSION THREAD

**Discussion by ian** [family/connectkit#39] [Q&A] [answered]
Just wondering if it's on the roadmap to migrate to wagmi 0.7.

**Comment by jylin** [comment]
Can't install this with newer versions of wagmi.

```npm ERR! While resolving: connectkit@1.0.0
npm ERR! Found: wagmi@0.8.2
npm ERR! node_modules/wagmi
npm ERR!   wagmi@"^0.8.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer wagmi@"^0.6.0" from connectkit@1.0.0
npm ERR! node_modules/connectkit
npm ERR!   connectkit@"^1.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: wagmi@0.6.8
npm ERR! node_modules/wagmi
npm ERR!   peer wagmi@"^0.6.0" from connectkit@1.0.0
npm ERR!   node_modules/connectkit
npm ERR!     connectkit@"^1.0.0" from the root project
```

**Comment by lochie** [comment]
ConnectKit hasn't yet been updated to work perfectly with the recent wagmi `0.8.x` update just yet. They've [dropped support for CommonJS modules](https://wagmi.sh/docs/migration-guide#commonjs-support-dropped), which is something we need to update here. We're hoping to get this update out asap.

I suggest downgrading wagmi to `0.6.x` (the last wagmi compatible ConnectKit release) until we have that update ready (hopefully within the next day or two)

**Reply by johnpaulcas** [reply]
hi @lochie , any update on this?

**Reply by lochie** [reply]
@johnpaulcas We've just released ConnectKit version 1.1.0 which works with wagmi 0.8.x! ✨

**Comment by lochie** [comment, marked as answer]
@ian @jylin ConnectKit version 1.1.0 now works with the latest version of wagmi 0.8.x. ☺️

**Reply by ian** [reply]
Great news! Thanks for the hustle @lochie
